### DISCUSSION TITLE
Shouldn't README.md be updated to show "v4"?

### DISCUSSION THREAD

**Discussion by blu3mania** [actions/checkout#1449] [General] [answered]
It currently is still showing "v3". This is especially an issue since "show-progress" option is only introduced in v4.

**Comment by patrick-stephens** [comment]
Yeah little confusing that and seems to be causing issues: https://github.com/actions/checkout/issues/1446

**Comment by illrill** [comment]
Also, perhaps a good idea to write explicitly what is not backwards compatible (given that it's released as a major).

**Comment by vanZeben** [comment, marked as answer]
Closed in https://github.com/actions/checkout/pull/1452
